src/components/TextArea/index.js

```diff
@@ -34,8 +34,8 @@ const TextareaField = glamorous.textarea(
       outlineOffset: 0
     }
   },
-  ({ errorText }) => ({
-    border: errorText ? `4px solid ${ERROR_COLOUR}` : `2px solid ${BLACK}`
+  ({ error }) => ({
+    border: error ? `4px solid ${ERROR_COLOUR}` : TextareaField.border
```

Comment:
I think

```
 `4px solid ${ERROR_COLOUR}` : undefined
```

would be best here, as it defaults it to whatever has been set above